src/server/pps/cmds/cmds.go

```diff
@@ -438,6 +439,24 @@ each datum.`,
 		follow      bool
 		tail        int64
 	)
+
+	// prettyLogsPrinter helps to print the logs recieved in different colours
+	prettyLogsPrinter := func(message string) {
+		informationArray := strings.Split(message, " ")
+		debugLevel := strings.ToLower(informationArray[1])
+		if debugLevel == "info" {
+			color.Set(color.FgGreen)
+		} else if debugLevel == "warning" {
+			color.Set(color.FgYellow)
+		} else if debugLevel == "error" {
+			color.Set(color.FgRed)
+		} else {
+			color.Set(color.FgWhite)
```

Comment:
In the case of the level not being determinable, just don't set a color -- that way it'll print with the default terminal color